$OpenBSD: patch-MCONFIG_in,v 1.4 2013/06/21 22:35:51 sthen Exp $

Statically link against e2fsprogs libcom_err to avoid the need to install it.
Avoids conflict with libcom_err from kerberosV.

--- MCONFIG.in.orig	Wed Jun 19 15:04:08 2013
+++ MCONFIG.in	Wed Jun 19 15:04:57 2013
@@ -89,7 +89,7 @@ MKINSTALLDIRS = @MKINSTALLDIRS@
 #
 LIB = $(top_builddir)/lib
 LIBSS = $(LIB)/libss@LIB_EXT@ @PRIVATE_LIBS_CMT@ @DLOPEN_LIB@
-LIBCOM_ERR = $(LIB)/libcom_err@LIB_EXT@ @PRIVATE_LIBS_CMT@ @SEM_INIT_LIB@
+LIBCOM_ERR = $(LIB)/libcom_err@STATIC_LIB_EXT@ @SEM_INIT_LIB@
 LIBE2P = $(LIB)/libe2p@LIB_EXT@
 LIBEXT2FS = $(LIB)/libext2fs@LIB_EXT@
 LIBUUID = @LIBUUID@ @SOCKET_LIB@
@@ -97,7 +97,7 @@ LIBQUOTA = @STATIC_LIBQUOTA@
 LIBBLKID = @LIBBLKID@ @PRIVATE_LIBS_CMT@ $(LIBUUID)
 LIBINTL = @LIBINTL@
 DEPLIBSS = $(LIB)/libss@LIB_EXT@
-DEPLIBCOM_ERR = $(LIB)/libcom_err@LIB_EXT@
+DEPLIBCOM_ERR = $(LIB)/libcom_err@STATIC_LIB_EXT@
 DEPLIBUUID = @DEPLIBUUID@
 DEPLIBQUOTA = @DEPSTATIC_LIBQUOTA@
 DEPLIBBLKID = @DEPLIBBLKID@ @PRIVATE_LIBS_CMT@ $(DEPLIBUUID)
